<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="b_srcM">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбор Ката</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_srcW">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбор Лога</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_fld">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>741</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Кудыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_start_and_done">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>901</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Запустить компоновку квитанций</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_srcM">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>путь до мэк</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_srcW">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>путь до водотеплухи</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_fld">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>путь до результатом доволен</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_chekMEK">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>чекМЭК(Отчёт№1)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumain">
    <property name="title">
     <string>main</string>
    </property>
   </widget>
   <addaction name="menumain"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
